lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -154,6 +154,12 @@ sub _get_ptero_lsf_parameters {
     $set_lsf_option->('queue', $attributes{lsfQueue});
     $set_lsf_option->('projectName', $attributes{lsfProject});
 
+    my $default_job_group = join('/',
```

Comment:
I didn't read the set_lsf_option closely enough to notice that it would not overwrite previous values.